# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/angle/checkout/out/gen/angle/id/commit.h
# Commit: 754436d8b3a2
# Full Hash: 754436d8b3a2023b90eb247b3a70335e74f2fdf6
# Author: Jeff Gilbert <jgilbert@mozilla.com>
# Date: 2020-05-09 09:14:23
# Regressor Bug: 1633628
# File Overlap Count: 3
# Description:
#   Bug 1633628 - Vender: Don't use ClearView if we previously used dual source blending on Intel gen6. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D74137
# ==============================================================================

diff -r f8fd6f0a5102 -r 754436d8b3a2 gfx/angle/checkout/out/gen/angle/id/commit.h
--- a/gfx/angle/checkout/out/gen/angle/id/commit.h	Fri May 08 18:39:28 2020 +0000
+++ b/gfx/angle/checkout/out/gen/angle/id/commit.h	Fri May 08 19:28:53 2020 +0000
@@ -1,3 +1,3 @@
-#define ANGLE_COMMIT_HASH "3885ee272685"
+#define ANGLE_COMMIT_HASH "8df54289d717"
 #define ANGLE_COMMIT_HASH_SIZE 12
-#define ANGLE_COMMIT_DATE "2020-01-22 18:48:23 -0800"
+#define ANGLE_COMMIT_DATE "2020-05-06 11:04:42 -0700"